### ISSUE TITLE
Consistency only works with two members...

### ISSUE THREAD

**Issue by hulucc** [buraksezer/olric#92]
```
	c := config.New("lan")
	c.BindAddr = "0.0.0.0"
	c.MemberlistConfig.BindAddr = "0.0.0.0"
	c.ReadRepair = true
	if os.Getenv("OLRIC_DISCOVERY_PROVIDER") == "k8s" {
		ns := os.Getenv("OLRIC_DISCOVERY_NAMESPACE")
		labelname := os.Getenv("OLRIC_DISCOVERY_LABEL_NAME")
		labelvalue := os.Getenv("OLRIC_DISCOVERY_LABEL_VALUE")
		labelSelector := fmt.Sprintf("%s=%s", labelname, labelvalue)
		c.ServiceDiscovery = map[string]interface{}{
			"plugin":   &discovery.CloudDiscovery{},
			"provider": "k8s",
			"args":     fmt.Sprintf("namespace=%s label_selector=\"%s\"", ns, labelSelector),
		}
	}
```
After rolling update on k8s, run `olric-cli -a $HOSTNAME:3320 -dmap status.health -c "put key 1"`
The result from four members is not consistent without any node absent. And set `ReadRepair` is not helping.
![image](https://user-images.githubusercontent.com/4343900/121132861-3f58c880-c864-11eb-8ced-6a656a6a84a1.png)

**Comment by buraksezer** [comment]
Hi @hulucc 

I have just released [Olric v0.4.0-beta.5](https://github.com/buraksezer/olric/releases/tag/v0.4.0-beta.5) to fix that issue. Could you please your case with the latest version?

**Comment by buraksezer** [comment]
Hi @hulucc 

I have just released [Olric v0.4.0-beta.6](https://github.com/buraksezer/olric/releases). It now works more reliable. It's highly recommended to upgrade to the latest beta. 

I'm closing this issue now. Please don't hesitate to open an issue, if you have any further problems.
